repose-aggregator/components/filters/client-auth/src/test/java/org/openrepose/filters/clientauth/openstack/OpenStackAuthenticationHandlerTest.java

```diff
@@ -838,12 +840,12 @@ public void shouldPassValidCredentials() throws Exception {
             token.getTenant().setId("12345");
             token.getTenant().setName("12345");
             token.setExpires(dataTypeFactory.newXMLGregorianCalendar((GregorianCalendar) expires));
-            authResp.setToken(token);
-            authResp.setUser(user);
-            authResp.getUser().setRoles(getTwoRoles());
-            authResp.getToken().setId("tokentokentoken");
+            authResponse.setToken(token);
+            authResponse.setUser(user);
+            authResponse.getUser().setRoles(getTwoRoles());
+            authResponse.getToken().setId("tokentokentoken");
 
-            when(authService.validateToken(anyString(), anyString())).thenReturn(authResp);
+            when(authService.validateToken(anyString(), anyString())).thenReturn(authResponse);
             final FilterDirector director = handler.handleRequest(request, response);
             assertEquals("Auth component must pass valid requests", FilterAction.PASS, director.getFilterAction());
         }
```

Comment:
This was the only test that named the response differently so I updated it for consistency.